classes/sota.bbclass

```diff
@@ -29,7 +30,7 @@ INITRAMFS_FSTYPES ??= "${@oe.utils.ifelse(d.getVar('OSTREE_BOOTLOADER', True) ==
 
 # Please redefine OSTREE_REPO in order to have a persistent OSTree repo
 OSTREE_REPO ?= "${DEPLOY_DIR_IMAGE}/ostree_repo"
-OSTREE_BRANCHNAME ?= "${MACHINE}"
+OSTREE_BRANCHNAME ?= "${HARDWARE_ID}"
```

Comment:
I think `SOTA_HARDWARE_ID` might be slightly better. I'm debating whether `PRIMARY` is a good word to include as well (or instead of `SOTA`), but that seems like overkill and might not even be accurate; we can also bitbake secondaries.